src/Symfony/Component/HttpClient/Response/ResponseTrait.php

```diff
@@ -235,6 +235,15 @@ private static function addResponseHeaders(array $responseHeaders, array &$info,
         }
     }
 
+    private static function handleContentType(&$content, string $contentType)
+    {
+        $contentType = substr($contentType, 0, strpos($contentType, ';'));
+        if (1 === preg_match('/(text\/[a-z]+|[a-z]+\/(json|xml)$)/', $contentType) && null === $content) {
+            $options['buffer'] = true;
```

Comment:
undefined variable - but not need to set this here, this no need for passing it by ref (this was the intent I suppose)

Reply:
Removed the unused variable